CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Development dependencies install via pip: `pip install "merlinwf[dev]"`
 - `merlin status <yaml spec>` that returns queues, number of connected
   workers and number of unused tasks in each of those queues
+- The step/run field `countdown`, which adds a delay in seconds to retries.
```

Comment:
I think this name should be more descriptive like `retry_delay`

Reply:
For some reason that's the name celery uses but we can change it.

Reply:
or retry_countdown would be good too